presto-accumulo/pom.xml

```diff
@@ -363,6 +367,12 @@
             <artifactId>jackson-databind</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
```

Comment:
For Presto, all dependency versions should go in the root POM's `dependencyManagement` section if possible.